src/volunteer-pages/addVolunteer/index.tsx

```diff
@@ -0,0 +1,85 @@
+import style from './style.module.css'
+import React, { FunctionComponent, useState } from 'react'
+import axios from 'axios'
+import { useHistory } from 'react-router-dom'
+
+interface AddVolunteerInfo {
+  email: string
+  password: string
+  confirmPassword: string
+}
+
+const AddVolunteer: FunctionComponent = () => {
+  const [newVolunteer, setNewVolunteer] = useState<AddVolunteerInfo>({
+    email: '',
+    password: '',
+    confirmPassword: ''
+  })
+
+  const [error, setError] = useState('')
+
+  const history = useHistory()
+
+  const onChange =
+    (stateKey: string) =>
+    ({ target }: { target: HTMLInputElement }) => {
+      setNewVolunteer({ ...newVolunteer, [stateKey]: target.value })
+    }
+
+  const matchedPassword = () => newVolunteer.password === newVolunteer.confirmPassword
+
+  const onClick = () => {
+    setError('')
+
+    if (!matchedPassword()) {
+      setError('passwords dont match, check and start again')
+      return
+    }
+    axios
+      .post(process.env.REACT_APP_API_URL + '/Volunteer/add', newVolunteer)
```

Comment:
it's a little bit annoying to have  `'/Volunteer/add'` here and  ` '/volunteer/signin'` in the last file 😅, for consistency keep it all lower cased

Reply:
alrighty